examples/hashsmt/sha1_inversion.cpp

```diff
@@ -54,19 +54,14 @@ int main(int argc, char* argv[]) {
     ofstream output(argv[2]);
     output << expr::ExprSetDepth(-1) << language::SetLanguage(language::output::LANG_SMTLIB_V2);
     output << SetBenchmarkLogicCommand("QF_BV") << endl;
-    output << SetBenchmarkStatusCommand(SMT_SATISFIABLE) << endl;
+
+    hashsmt::Word message(msg.size()*8, "x");
 
     // Make the variables the size of the string
     hashsmt::cvc4_uchar8 *cvc4input = new hashsmt::cvc4_uchar8[msgSize];
```

Comment:
Suggestion: This could also just be a vector, like that we don't need the `delete[]` at all.